internal/rand/rand_bench_test.go

```diff
@@ -0,0 +1,201 @@
+//
+// Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package rand
+
+import (
+	"math"
+	"testing"
+)
+
+func BenchmarkUint32(b *testing.B) {
+	b.ReportAllocs()
```

Comment:
Sorry I think we need to call `ResetTimer()` before the benchmark runs.
```suggestion
	b.ReportAllocs()
	b.ResetTimer()
```